ts/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import * as anchor from "@project-serum/anchor";
```

Comment:
We should probably group all TS packages together under `packages-ts` like we currently do on `chainlink-terra`.

Reply:
Actually, the other way around. Gauntlet packages on Terra should be split and moved to a `gauntlet/` subfolder so a 3rd party can build just the relevant packages without Gauntlet. With `gauntlet-core` being private, trying to build a workspace with this package + `gauntlet-solana` will fail.

Reply:
`gauntlet-core` is now a public package.